x/cosmwasmpool/model/store_model.go

```diff
@@ -44,7 +44,7 @@ func (p CosmWasmPool) GetAddress() sdk.AccAddress {
 }
 
 func (p CosmWasmPool) GetId() uint64 {
-	panic("CosmWasmPool.GetId not implemented")
+	return p.PoolId
 }
```

Comment:
Highlighting that we had to implement this method, otherwise when we call SetPool in InitGenesis with a CosmWasmPool instead of a Pool, we need access to the GetId method